src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java

```diff
@@ -95,7 +96,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     String lectureName = request.getParameter(PARAM_NAME);
     Entity lectureEntity = LectureUtil.createEntity(lectureName, videoUrl, videoId.get());
     datastore.put(lectureEntity);
-    initializeTranscript(lectureEntity);
+    String transcriptLanguage = request.getParameter(PARAM_LANGUAGE);
```

Comment:
Can you add a check for this this to the `validatePostRequest` call?

Reply:
I think it would be better not to for now since the language input is not required. If there isn't a language, the parsing error is caught in the try-catch block.  I updated it to use optional though so `transcriptLanguage` wouldn't be null if a language isn't selected.